layouts/default-max.vue

```diff
@@ -1,20 +1,29 @@
 <template>
   <!-- tabindex is needed to allow hiding the menu in iOS Safari -->
   <div tabindex="-1">
-    <header id="navigation">
-      <TheMenu @change-visibility="isMenuShown = $event === 'shown'" />
-    </header>
-    <nuxt />
+    <qiskit-ui-shell variant="hide-account" @on-click="onClick" />
+    <div class="main-container">
+      <nuxt />
+    </div>
     <PageFooter theme="light" />
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
+import '@qiskit/web-components/components/ui-shell'
 
 @Component
 export default class MaxLayout extends Vue {
-  isMenuShown: boolean = false
+  onClick (e: CustomEvent) {
+    this.$trackClickEvent(`${e.detail?.label | e.detail?.url}`, 'navbar')
```

Comment:
The structure in which we should be passing the information to Segment is different.

In `constants/menuLinks.ts` on line 119 we can see how it is currently defined.

For the "Overview" link for instance, we should pass `"overview"` as `cta` and `"menu"` as `location`.

We need to keep the current approach to avoid disruptions in our analytics.

Reply:
Thanks @eddybrando! I'll take a look again..from what I understood, we aren't currently sending analytics data in our navbar, but maybe I misunderstood, not sure if you also have some insight @techtolentino 

For the "dropdown" portions in the new ui-shell, I think we chatted about this yesterday Eddy, but we didn't add any event dispatches for toggling the dropdown, only when the actually link is clicked. let me know if you think that needs to be updated

Reply:
in my testing, tracking the clicks in the `navbar` doesn't seem to be working


See here in the screen capture, we're only getting Segment page loaded events... 

https://user-images.githubusercontent.com/6276074/190444399-8b96038c-692f-4da6-acb1-3460c1016dc7.mov

Reply:
resolved by https://github.com/Qiskit/qiskit.org/pull/2784/commits/7302c36eb95643fa7d46e576a660b1ceb8af6416